modules/creating-runtimeclass.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * windows_container_workloads/scheduling-windows-workloads.adoc
+
+[id="creating-runtimeclass_{context}"]
+= Creating a `RuntimeClass` to encapsulate scheduling mechanisms
+
+Using a `RuntimeClass` simplifies the use of scheduling mechanisms like taints and tolerations; you deploy a `RuntimeClass` that encapsulates your taints and tolerations and then apply it to your pods to schedule them to the appropriate node. Creating a `RuntimeClass` is also necessary in clusters that support multiple operating system (OS) variants.
+
+.Procedure
+
+. Create a `RuntimeClass` object YAML file. For example, `runtime-class.yaml`:
++
+[source,yaml]
+----
+apiVersion: node.k8s.io/v1beta1
+kind: RuntimeClass
+metadata:
+  name: <runtime_class_name> <1>
+handler: 'docker'
+scheduling:
+  nodeSelector: <2>
+    kubernetes.io/os: 'windows'
+    kubernetes.io/arch: 'amd64'
+    node.kubernetes.io/windows-build: '10.0.17763'
+  tolerations: <3>
+  - effect: NoSchedule
+    key: os
+    operator: Equal
+    value: "windows"
```

Comment:
I need to change to value: "Windows", with capitalized "w", or else schedule Windows workload failed with:
Warning  FailedScheduling  7s          0/8 nodes are available: 1 node(s) had taint {os: Windows}, that the pod didn't tolerate, 7 node(s) didn't match node selector.